### ISSUE TITLE
ZCash Debit Card 

### ISSUE THREAD

**Issue by CryptoFarm7** [ZcashFoundation/GrantProposals-2017Q4#9]
_We own ZCashNow, CryptoFarm7 and ZCashNowPool_

In connection with the ZCash Foundation Grants Call for Proposals (Q4 2017) we wish to apply for a grant to build a ZCash Debit Card. 

**Motivation and overview**
We are working to build a ZCash Debit card to get more people involved with the ZCash Community.   Many of our customers see the need for such a prepaid card because they currently must transfer their ZCash to Bitcoin to send to their Bitpay Card.  This makes ZCash seem second class, why should they have to convert their coins to bitcoin?  We want to see people use the ZCash Network independently of Bitcoin. 

**Technical approach**

1.	Create an easy to use GUI interface
2.	Build a physical ZCash Debit card that allows customer to transfer ZCash to their Prepaid ZCash Debit card which allows the customer to use anywhere VISA and Mastercard are accepted
Team background and qualifications
Our President has been an entrepreneur since age 16 and has built many multi-million-dollar companies over the years.  Our CTO has extensive background in blockchain technologies, has built mining pools, crypto coins, developed prepaid debit cards and has been a professional programmer for over 20 years working for fortune 500 companies.  Our COO has built many large companies and has been an entrepreneur his entire life.  Other members of our team bring capital, connections and expertise from all areas of business, finance and legal.  We have a total of around $500,000 already invested in ZCash projects, including a mining pool a commercial ZCash mining farm and have pledged up to $2,500,000 to Metropolitan Commercial Bank as collateral if we get this grant to build the ZCash Debit Card. 

**Evaluation plan**

Once the features are implemented (at each stage/feature), they may be evaluated by the Grant Review Committee and/or other community members.  We will also run a contest for the community to help design the ZCash Debit Card.  The measure of success will be satisfied when ZCash users have access to an easy to use ZCash Debit Card. 

**Security considerations**

We will be working with Metropolitan Commercial Bank, VISA and a whole host of partners where our focus will be on security and compliance.  We have already formed partnerships with these companies and have signed agreements in place ready to start development. 

**Schedule**

The ZCash Debit card will be ready for market on January 21, 2018.   We will have the card built and complete by December of 2017, however; VISA has a freeze from November 16 to January 18 each year on release of new prepaid debit cards.  This freeze is due to holiday rush and volume of cards being used during this time.  Because of this Freeze, we will have extra time to test and refine our ZCash Debit card to ensure a full functioning flawless card upon launch date.

_Budget and justification_
$5,000 Expedite Fee – Galileo 
$25,000 Bank Fee – Metropolitan Commercial Bank
$30,000 Card Production – Galileo
$20,000 Marketing & Distribution
($2.5 million 2 Year Loan Agreement with Bank - Metropolitan Commercial Bank)

We have all agreements signed and in place with all necessary parties to make this card a reality, we are pledging the $2.5 Million in escrow money to the bank for the 2-year loan agreement upon receipt of the grant to develop the ZCash Debit Card. 

www.ZCashNow.com     www.CryptoFarm7.com    www.ZCashNowPool.com 
https://www.facebook.com/ZCashNow    https://www.facebook.com/CryptoFarm7/

**Comment by mineZcash** [comment]
I assume you have documentation that shows you have permission to sell and use the Zcash name and logo? Otherwise you are likely in violation of Zerocoin Electric Coin Company trademarks.

https://z.cash/trademark-policy.html

**Comment by CryptoFarm7** [comment]
Yes, we got permission back in Spring of this year.

Sincerely,
Chad Atkins

**Comment by tromer** [comment]
@CryptoFarm7, is the proposed debit card essentially different than Uquid's (https://uquid.com/altcoin-debit-card)?

Regarding trademarks, does the scope of the permission you received include branded debit cards?

**Comment by CryptoFarm7** [comment]
Our Card would be very similar to the uquid card, however; in the USA you can not have a physical uquid debit card, whereas you our card will be specifically for the USA.   We do not intend to brand it ZCash debit card, we will use our company name just as uquid has done but will be for only ZCash Cryptocurrency.

**Comment by mineZcash** [comment]
Since in the USA Bitpay card and Coinbase Shift card require full identification to comply with KYC & AML laws (and a new card will likely be required to do the same) there will be no privacy gains by using Zcash as opposed to Bitcoin for loading a card, is that correct?

**Comment by CryptoFarm7** [comment]
Yes we plan to be completely compliant with all USA rules and regulations so the privacy information will be required.  Our goal is to save ZCash users Exchange Fees, currently in order for a ZCash user to get US Dollars in the USA, the ZCash User must first convert their ZCash to Bitcoin, then Send their bitcoin to their Bitpay Debit card, which causes the ZCash user to incur Additional Exchange fees.

**Comment by tromer** [comment]
@CryptoFarm7, what experience does you team have with the operational aspect of issuing debit cards? KYC/AML compliance, customer (and merchant) support in this kind of market, fraud detection+recovery, etc.

**Comment by CryptoFarm7** [comment]
My lead programmer currently works for a fortune 500 company with over 20 years experience.  He builds solutions of all kinds for other fortune 500 companies.   He has already built 5 other debit cards for corporate clients.  We are also partnered with Galileo https://galileoprocessing.com/ and Metropolitan Commercial Bank, we are using their team to support and help us build to get a complete secure compliant solution ready for market.  for merchant and customer support we will be utilizing Galileo to provide a seamless support of the launched card.  We are not rushing the process, we are working through the freeze process in order to ensure we have time to test and retest before launch.

**Comment by tromer** [comment]
Let's discuss privacy.
1. What parties would get access to card owner's personal information?
2. What parties would get access to transaction history?
3. What limits/safeguards/accountability are there (or can be placed) on these parties' use of the information?
4. Is it possible to legally issue and operate such cards without keeping users' personal information on record? (See the "how to tip a busker in Zcash" dscussion that starts here: https://chat.zcashcommunity.com/channel/the-zcash-foundation?msg=N89B7SksPmsBhtTTx)

**Comment by cryptoking7** [comment]
tromer, sorry for the delay in getting back with a response to your questions. As Chad mentioned earlier in the thread we have a partner, Galileo Processing, to assist us in making sure we are compliant and we had hoped to be able to get an answer today but both Ben and Chad are traveling this weekend and I wanted to make sure I had their input before responding. I did not want to leave you hanging on why we did not respond to your request but I expect Ben with Galileo to return to the office Monday and as soon as he and the Bank communicate I am sure we will have answers for you. Thanks for your questions and if you have additional requests or questions we encourage those as well, thanks.

Daniel Francis
Cryptofarm7

**Comment by CryptoFarm7** [comment]
@tromer   Very sorry for the delay in response, I am in travels,  Below are the questions answered, I hope this helps.  

1.       What parties would get access to card owner's personal information? Normally the full personal information (Name, SSN, Address, Date of Birth, Card Number) are securely stored by Galileo. We can only share the full information with ZCash if the cardholder has given consent and ZCash is PCI compliant. We do have the ability to provide ZCash with masked information. We also employ a proxy number that is used for all of our communication. 
2.       What parties would get access to transaction history? The issuing bank and Galileo will have access to this information by default. This information will also be available to ZCash with proper cardholder consent. The cardholder gives consent during the application process. 
3.       What limits/safeguards/accountability are there (or can be placed) on these parties' use of the information? We are open to working with ZCash and the issuing bank to define how this should be structured. Typically these term are outlined in our service agreements. 
4.       Is it possible to legally issue and operate such cards without keeping users' personal information on record? This is usually a question for the issuer, but I don’t know of any issuer that will allow for a cardholder to operate on their system without the personal information being on record. There are all kinds of laws that require them to know the customer and be accountable for the cardholder’s purchasing behavior.

**Comment by acityinohio** [comment]
Every informal proposal has multiple reviews by the review committee. The reviews are being collected and discussed in a private google doc (the 5 reviewers all have edit access to it, no one else can view it). By way of early, informal feedback, the reviewers would like to inform **you that your proposal is not a leading candidate for funding in this round.**

You may still choose to submit a full proposal by **October 6th**, and it will be evaluated. But if it’s substantially the same as discussion so far reflects, then we suspect it is unlikely to be chosen for funding; and if it isn't, then we encourage you to post a draft as early as possible, to allow for community feedback. Regardless of your choice we thank you for participation thus far.
